common/utils/dates.ts

```diff
@@ -19,13 +19,11 @@ export function today(): Date {
 }
 
 export function isPast(date: Date): boolean {
-  const now = new Date();
-  return date < now;
+  return date < today();
```

Comment:
I changed this because I thought I might want to mock `today()` in the tests (which we do elsewhere), but it turns out I need to mock `isFuture()` instead.